### ISSUE TITLE
AttributeError: 'SessionInfo' object has no attribute 'report_run_count'

### ISSUE THREAD

**Issue by thanyi** [whitphx/streamlit-video-chat-example#51]
# summary
after install in terminal: 
```
pip install streamlit streamlit-webrtc streamlit-server-state opencv-python-headless
```
error in terminal：
```
streamlit run app_mcu_filters.py
```

# error message
```
[2022-02-28 09:22:40,283] WARNING from streamlit_webrtc.component in C:\Users\ethanyi\anaconda3\envs\pytorch\lib\site-packages\streamlit_webrtc\component.py:342: The argument client_sett
ings is deprecated. Use rtc_configuration and media_stream_constraints instead.
2022-02-28 09:22:40.287 Traceback (most recent call last):
  File "C:\Users\ethanyi\anaconda3\envs\pytorch\lib\site-packages\streamlit\script_runner.py", line 430, in _run_script
    exec(code, module.__dict__)
  File "D:\DeepFakeProject_in_D\deepfake_project\eliminate_project\video_chat\demo_online2\streamlit-video-chat-example\app_mcu_filters.py", line 271, in <module>
    main()
  File "D:\DeepFakeProject_in_D\deepfake_project\eliminate_project\video_chat\demo_online2\streamlit-video-chat-example\app_mcu_filters.py", line 193, in main
    self_ctx = webrtc_streamer(
  File "C:\Users\ethanyi\anaconda3\envs\pytorch\lib\site-packages\streamlit_webrtc\component.py", line 420, in webrtc_streamer
    run_count = session_info.report_run_count if session_info else None
AttributeError: 'SessionInfo' object has no attribute 'report_run_count'
```

**Comment by whitphx** [comment]
Thank you for the report.

This problem began to occur since Streamlit v1.6, and the fixed version of `streamlit-webrtc`, v0.35.2 is already released (Ref: https://github.com/whitphx/streamlit-webrtc/issues/709).

I will update the `requirements.txt` in this repo.

**Comment by whitphx** [comment]
#57
